### PR TITLE
Fixing issue #1 of (TS2802)

### LINKED ISSUE #1: Down level Iteration - #TS2802
image

Type 'IterableIterator<[string, string]>' can only be iterated through when using the '--downlevelIteration' flag or with a '--target' of 'es2015' or higher.

in this line: https://github.com/handyscript/handyutility/blob/b45b7fac31ba0449381dd1b3b850c80a297a1a24/utils/handy.ts#L24

### PR DESCRIPTION
**Pull request by ilorez**
the Array.from used to convert the iterable url.searchParams.entries() to an array before iterating over it using forEach. This change allows the code to work without requiring the --downlevelIteration flag or a higher target version.
I hope that fixed the probleme

### CODE REVIEW

File: utils/handy.ts [hunk 1]
```diff
@@ -3,11 +3,12 @@ import axios from "axios";
 
 /**
  * Retrieve the IP address of the current machine
- * @returns {Promise<string> as IP address}
+ * @returns {Promise<string>} as IP address
  */
 export async function IPify(): Promise<string> {
   try {
-    return await (await axios.get("https://api.ipify.org?format=json")).data.ip;
+    const response = await axios.get("https://api.ipify.org?format=json");
```
Comment (vvhybe) [inline utils/handy.ts:10]: Don't stick to code that has not been referenced in the issue ( #1 )

File: utils/handy.ts [hunk 2]
```diff
@@ -21,8 +22,9 @@ export async function IPify(): Promise<string> {
  */
 export function searchParamsSerializer(url: URL): Record<string, string> {
   const params: Record<string, string> = {};
-  for (const [key, value] of url.searchParams.entries()) {
+  // Convert the entries to an array and iterate over it
+  Array.from(url.searchParams.entries()).forEach(([key, value]) => {
```
Comment (vvhybe) [inline utils/handy.ts:26]: Did you test it ??

**Review by vvhybe — CHANGES_REQUESTED** [review]

File: utils/handy.ts (hunk 1 above)
Comment (ilorez) [reply utils/handy.ts:10]: Okay!

File: utils/handy.ts (hunk 2 above)
Comment (ilorez) [reply utils/handy.ts:26]: i found same issue with this solution in other place you can test it and see if it work correctly...

Comment (vvhybe) [reply utils/handy.ts:26]: Now that's cool I'll be implementing it soon

Comment (vvhybe) [reply utils/handy.ts:26]: i fix it, using direct push to the main:
https://github.com/handyscript/handyutility/blob/4b7f38ec4546c8d10034c269e0a0a319706a0471/utils/handy.ts#L24

**Comment by vvhybe** [comment]
https://github.com/handyscript/handyutility/blob/4b7f38ec4546c8d10034c269e0a0a319706a0471/utils/handy.ts#L24

File: utils/handy.ts (hunk 2 above)
Comment (ilorez) [reply utils/handy.ts:26]: Great JOB, Keep going Bro.

**Pull request closed without merging**